apps/smbmount.sh

```diff
@@ -0,0 +1,288 @@
+#!/bin/bash
+
+# T&M Hansson IT AB © - 2020, https://www.hanssonit.se/
+
+# shellcheck disable=2034,2059
+true
+# shellcheck source=lib.sh
+. <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
+
+# Check for errors + debug code and abort if something isn't right
+# 1 = ON
+# 0 = OFF
+DEBUG=0
+debug_mode
+
+# Check if root
+root_check
+
+# install cifs-utils
+install_if_not cifs-utils
+
+# secure fstab
+if [ "$(stat -c %a /etc/fstab)" != "600" ]
+then
+    chmod 600 /etc/fstab
+fi
+
+# functions
+add_mount() {
+# check if mounting slots are available
+if grep -q /mnt/smbshares/1 /etc/fstab && grep -q /mnt/smbshares/2 /etc/fstab && grep -q /mnt/smbshares/3 /etc/fstab
```

Comment:
Please make `/mnt/smbshares` a variable if we decide to change the name later on: `$SMBSHARES`.

Reply:
I have done this. But I had to make to variables, because sed didn't like the formatting.